src/game/ctrl.rs

```diff
@@ -129,9 +129,9 @@ impl GameController {
     }
 
     pub fn poll_analog2d(&self, motion: &Analog2d) -> Vec2f {
-        match motion {
-            &Analog2d::Mouse(sensitivity) => self.mouse_rel * sensitivity,
-            &Analog2d::Gestures(
+        match *motion {
```

Comment:
Yes, this is only for readability. The generated code should be exactly the same.